INSTALL.md

```diff
@@ -101,13 +101,17 @@ Create config file, based on `dendrite-config.yaml`. Call it `dendrite.yaml`. Th
 ## Starting a monolith server
 
 It is possible to use 'naffka' as an in-process replacement to Kafka when using
-the monolith server. To do this, set `use_naffka: true` in `dendrite.yaml`.
+the monolith server. To do this, set `use_naffka: true` in `dendrite.yaml`. 
+
+In the `dendrite.yaml` file, one must also uncomment the necessary line (related to naffka) to use naffka.
```

Comment:
Could you please name the "necessary line" for clarity here?

Reply:
Sure, will do that!